03-objects-and-composition/exercise/07.js

```diff
@@ -0,0 +1,25 @@
+function townsToJSON(array){
+  const [, ...items] = array
+  //  regexp to match all spaces and |
+  let regex = /\s?\|\s?/;
+  let res = [];
+  
+  for (let i = 0; i <items.length; i++) {
+    let [, town, latitude, longitude,] = items[i].split(regex);
+    latitude = Number(latitude);
+    longitude = Number(longitude);
+    let obj = {};
+    obj.Town = town;
+    // Помощ за тези 2 реда ако може
+    obj.Latitude = Math.round(latitude * 100) / 100;
+    obj.Longitude = Math.round(longitude * 100) / 100;
+    res.push(obj);
+  }
+
+  console.log(JSON.stringify(res));
+ 
```

Comment:
You can remove the empty lines and format the code always when writing code so to make it clearer. If you want cool helper for this you can checkout Prettier.